manager/integration/tests/common.py

```diff
@@ -4680,3 +4681,103 @@ def get_volume_running_replica_cnt(client, volume_name):  # NOQA
             client, volume_name, node.name, chk_running=True)
 
     return cnt
+
+
+def wait_for_orphan_count(client, number, retry_counts=120):
```

Comment:
I think we should organize common.py better based on different resources and util functions in the future for better maintenance. 

cc @yangchiu @cchien816